src/scripts/main.js

```diff
@@ -1,3 +1,81 @@
 'use strict';
 
-// write code here
+document.addEventListener('DOMContentLoaded', () => {
+  const table = document.querySelector('.field');
+  const appendRowButton = document.querySelector('.append-row');
+  const removeRowButton = document.querySelector('.remove-row');
+  const appendColumnButton = document.querySelector('.append-column');
+  const removeColumnButton = document.querySelector('.remove-column');
+
+  let rowCount = table.rows.length;
+  let colCount = table.rows[0].cells.length;
```

Comment:
If the table is initially empty, accessing `table.rows[0].cells.length` will throw an error. Consider adding a check to ensure the table has rows and cells before accessing these properties.